haskell/private/actions/compile.bzl

```diff
@@ -29,6 +29,8 @@ def _process_hsc_file(hs, cc, hsc_flags, hsc_file):
       (File, string): Haskell source file created by processing hsc_file and
          new import directory containing the produced file.
     """
+    if hs.toolchain.is_windows:
+        cc = "Bazel's CC shouldn't be used on Windows"
```

Comment:
Shouldn't this be a call to `fail()`? But also, where are we using "Bazel's CC" here? What does this refer to? the `CC` environment variable?

Reply:
It seems to override the function's `cc` argument. I think it's here to ensure that we don't actually use it.